carease/app/comparison/page.jsx

```diff
@@ -1,11 +1,98 @@
+import NumFour from '@/public/images/4.svg'
+import IPhoneFrame01 from '@/public/images/iPhoneFrame-01.svg'
+import IPhoneFrame02 from '@/public/images/iPhoneFrame-02.svg'
 
 const Comparison = () => {
   return (
-    <>  
-      {/* <Navbar />  */}
+    <>
       <Categories />
+
+      <section className='p-2 md:px-16'>
+          <h1 className='my-8 text-2xl font-bold'>Already know what you’re looking for ?</h1>
+          <div className='flex flex-col text-sm sm:text-lg md:justify-center text-center text-[#312C2C]'>
+            
+            <div className="w-full py-8 md:px-12 flex justify-between items-center">
+              <div className="h-[50px] flex items-center gap-4">                
+                <Image 
+                  src={NumOne}
+                  width={20}
+                  height={20}
+                  alt="numOne"  
+                  className="w-4 md:w-6"                
+                />                
+              
+                <h2>Search for cars that you’d like to compare</h2>
+                
+              </div>
+              
+              
+              <Image 
+                src={IPhoneFrame01}
+                width={190}
+                height={385}
+                alt="imgOne"
+                className="lg:mr-40"
+              />
+            </div>
+
+
+            <div className="w-full py-8 flex justify-around items-center">
+              
+              <Image 
+                src={IPhoneFrame02}
+                width={190}
+                height={385}
+                alt="imgOne"
+              />              
+
+              <div className="h-[50px] flex items-center gap-4">
+                <Image 
+                  src={NumTwo}
+                  width={20}
+                  height={20}
+                  alt="numTwo"
+                  className="w-4 md:w-6" 
+                />
+                <h2>Tap Compare. You’re done. No really, you are</h2>
+              </div>
+
+            </div>
+
+
+            <div className="w-full py-8 flex justify-around items-center">
+
+              <div className="md:px-12 pt-4 flex items-start">                
+                <Image 
+                  src={NumThree}
+                  width={20}
+                  height={20}
+                  alt="numThree"
+                  className="w-4 md:w-6" 
+                />
+                <h2>A huge comparison table comes up, allowing you to compare components and specs for each car.</h2>
+              </div>
+
+              <div className="md:px-12 pt-4 flex items-start">                
+                <Image 
+                  src={NumFour}
+                  width={20}
+                  height={20}
+                  alt="numFour"           
+                  className="w-4 md:w-6"        
+                />
+                <h2>Want to delve into a car more? Click on the boke’s  name to view-in-depth info, reviews, prices,specification, images and more.</h2>
```

Comment:
Change the name in the h2 with car's name instead of boke's name

Reply:
Changes done.